### DISCUSSION TITLE
A guest to speak about ARM64 improvements in .NET on ARM Innovation Coffee

### DISCUSSION THREAD

**Discussion by neon-sunset** [dotnet/runtime#94870] [Q&A] [answered]
Recently, Innovation Coffee podcast by ARM software developers community [has posted a video](https://www.youtube.com/watch?v=eZ0Sx57Io6w) discussing ARM64 improvements in Java.

It appears they are also happy to invite someone from .NET (compiler) team to talk about ARM64 improvements on our side of the lawn :)

The question was raised on The Arm Software Developers Discord server so I'm creating a discussion here.

**Comment by kunalspathak** [comment]
@JulieLeeMSFT

**Comment by mhall119** [comment]
Hi all, I'm the Arm Developer Evangelist who hosted that video.

If anybody can point me to PRs or docs about Arm optimizations in .Net, I'd like to put together some key things to talk about.

**Reply by kunalspathak** [reply]
Here are few blogs we published over the past few releases:

- https://devblogs.microsoft.com/dotnet/arm64-performance-in-net-5/
- https://devblogs.microsoft.com/dotnet/arm64-performance-improvements-in-dotnet-7/
- https://devblogs.microsoft.com/dotnet/this-arm64-performance-in-dotnet-8/

**Reply by saint4eva** [reply]
@tannergooding

**Reply by mhall119** [reply]
These blogs are great @kunalspathak! 

Would you be interested to come on our show to talk about this work? Our guest for this week has unfortunately had to reschedule, so I’m looking for somebody who can join us this Thursday at 5pm UTC. It would be very casual, we will prepare some questions and things to talk about ahead of time, but otherwise it’ll just be a natural conversation. 

Would you be available this week? If not, is there a Thursday in December where you could come on the show?

**Reply by kunalspathak** [reply]
Thanks @mhall119 for the invitation. This week I have some commitments, but I should be available on Dec 14th, if that works for you.

**Reply by mhall119** [reply]
@kunalspathak the 14th works for me! Please send me your email address to michael.hall <at> arm.com and I'll get you on our schedule.

Thanks again, I'm looking forward to having you on!

**Reply by kunalspathak** [reply]
Emailed you.

**Comment by mhall119** [comment, marked as answer]
@kunalspathak will be our guest on Innovation Coffee today! You will be able to watch the episode live, or the recording of it later, here on YouTube: https://www.youtube.com/watch?v=uCc1VcfVYSM
